samples/caf_sensor_manager/configuration/common/event_proxy_init.h

```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright (c) 2022 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#if CONFIG_EVENT_MANAGER_PROXY
+	int init_event_proxy(char *proxy_subscribe_event);
+#else
+	static inline int init_event_proxy(char *proxy_subscribe_event)
```

Comment:
I would better do `ifdef` in main instead of this. Its way more clear to me.

Reply:
It was there, @pdunaj asked me to hide it here.